src/plugins/upf/flowtable_init.c

```diff
@@ -137,6 +151,11 @@ flowtable_init (vlib_main_t * vm)
 	return error;
     }
 
+  vec_reset_length (name);
+  name=format (name, "/upf/flow_table_size%c", 0);
```

Comment:
1. you need to free name
2. since you use name only once, this could be simplified to
```
 name = format (0, "/upf/flow_table_size%c", 0);
```

Reply:
Got it